apps/codeforafrica/src/lib/api.netlify-cms/sections/getMeetOurTeam.js

```diff
@@ -0,0 +1,15 @@
+import { join } from "path";
```

Comment:
@kilemensi  are you referring to how the `index.js` file knows about different `sections`? e.g by 
```
import getMeetOurTeam from "./api.netlify-cms/sections/getMeetOurTeam";
```

Reply:
Yes @esirK . From API _user_ point of view, is there any reason why the 2 below are different?

```js
import getMeetOurTeam from "./api.netlify-cms/sections/getMeetOurTeam";
```
```js
import getProjects from "./api.netlify-cms/getProjects";
```

Reply:
It's not a JS thing, it's just section feels like an implementation detail here. Guiding principles, Partners, etc., are all sections in there respective pages but we don't include section in their import paths.

If I read the code correctly, the only real reason we have section here is because `MeetOurTeam` and `Hero` are part of file-based collections while the rest are folder-based collections. If this is true, then this **is** an implementation detail that users shouldn't really have to deal with.